src/Message/ExpressAuthorizeRequest.php

```diff
@@ -34,6 +91,36 @@ public function getData()
         $data['LOCALECODE'] = $this->getLocaleCode();
         $data['CUSTOMERSERVICENUMBER'] = $this->getCustomerServiceNumber();
 
+        $callback = $this->getCallback();
+
+        if (!empty($callback)) {
+            $data['CALLBACK'] = $callback;
+            // callback timeout MUST be included and > 0
```

Comment:
If the callback timeout is 0 it fails at PayPal's end. As the callback is optional there's no else branch if it's empty. So I'm using the `CALLBACK` option as a proxy for a `callbackEnabled` type of option.

Reference: https://developer.paypal.com/docs/classic/express-checkout/integration-guide/ECInstantUpdateAPI/
> Set CALLBACKTIMEOUT to the amount of time in seconds to process the callback. By default, CALLBACKTIMEOUT is 3. You can specify a value in the range of 1 to 6 inclusive.